mrtt-ui/src/views/Organizations.js

```diff
@@ -1,49 +1,64 @@
 import React, { useEffect, useState } from 'react'
 
 import { H4, H5 } from '../styles/typography'
+import { PaddedPageSection, PaddedPageTopSection } from '../styles/containers'
+import { toast } from 'react-toastify'
 import { UlAlternating } from '../styles/lists'
+import axios from 'axios'
 import language from '../language'
-import mockOrganizations from '../data/mockOrganizations'
-import { PaddedPageSection, PaddedPageTopSection } from '../styles/containers'
+import LoadingIndicator from '../components/LoadingIndicator'
+
+const organizationsUrl = `${process.env.REACT_APP_API_URL}/organizations`
 
 function Organizations() {
   const [yourOrganizations, setYourOrganizations] = useState([])
   const [otherOrganizations, setOtherOrganizations] = useState([])
-  useEffect(function formatOrganizations() {
-    setYourOrganizations(
-      mockOrganizations.filter((organization) => organization.isCurrentUserMember)
-    )
-    setOtherOrganizations(
-      mockOrganizations.filter((organization) => !organization.isCurrentUserMember)
-    )
+  const [isLoading, setIsLoading] = useState(true)
+  useEffect(function loadOrganizations() {
+    axios
+      .get(organizationsUrl)
+      .then(({ data }) => {
+        setIsLoading(false)
+        setYourOrganizations(data.filter((organization) => !!organization.isCurrentUserMember))
```

Comment:
the exclamation and double exclamation make this kind of hard to read/understand - is this the same as `organization?.isCurrentUserMember`

Reply:
ya, the !! probably isnt necessary here